modules/metastorage-server/src/test/java/org/apache/ignite/internal/metastorage/server/ValueConditionTest.java

```diff
@@ -57,4 +61,54 @@ public void ne() {
 
         assertTrue(cond.test(new Entry(KEY, VAL_2, 1, 1)));
     }
+
+    /**
+     * Tests that value is greater than another one.
+     */
+    @Test
+    public void gt() {
+        Condition cond = new ValueCondition(GREATER, KEY, VAL_1);
+
+        // byte[]{22} > byte[]{11}.
+        assertTrue(cond.test(new Entry(KEY, VAL_2, 1, 1)));
```

Comment:
I'd test some cases, when method returns false. At the moment, implementation `return true` will pass the test :)

Reply:
done